core-rust/state-manager/src/state_manager.rs

```diff
@@ -447,11 +439,12 @@ where
             let executable = prepared.get_executable();
 
             let next_epoch = {
```

Comment:
_(note from author)_

In a follow-up refactor I plan to pull the `next_epoch` tracking into `StateTracker`, it clicks nicely (see https://github.com/radixdlt/babylon-node/pull/474/files#diff-5230880996b770cf628e475ce1143be291e99fd9801f307fd1702a525c351aadR533).